README.md

```diff
@@ -6,6 +6,12 @@
 
 DDL module for Tarantool 1.10+
 
+The DDL module allows you to describe a data schema in the YAML format in a
+declarative way. It's alternative way to describe data schema using Lua,
+it is simpler to use, and you do not have to dive deep into Lua.
+DDL is a built-in Cartridge module. See more details about Tarantool's data mode
+in [documementation](https://www.tarantool.io/en/doc/latest/book/box/data_model/#data-schema-description-using-the-ddl-module).
```

Comment:
```suggestion
DDL is a build-in Cartridge module.
For more details about Tarantool's data model, refer to
[the documentation](https://www.tarantool.io/en/doc/latest/book/box/data_model/#data-schema-description-using-the-ddl-module).
```

Reply:
Thanks, Nick! The suggestion is committed.